supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -134,6 +134,8 @@ case class Run(runId: String,  jobDescriptor: BackendJobDescriptor, genomicsInte
           if (jesEndTime.offsetDateTime isAfter finalEventTime) jesEndTime else jesEndTime.copy(offsetDateTime = finalEventTime)
         case (Some(jesEndTime), None) => jesEndTime
         case (None, Some(finalEventTime)) => EventStartTime("cromwell poll interval", finalEventTime)
+        case (None, None) =>
+          throw new IllegalArgumentException("Both jesReportedEndTime and finalEventsListTime were None.")
```

Comment:
Let's tech talk any extra action we want here. Effectively all I've done with this is replace a `scala.MatchError`<sup>1</sup> with a `java.lang.IllegalArgumentException`. I saw a compiler warning and removed it with this "fix". I haven't actually changed the behavior/expectation for the callers who might trip over either the exception. I suspect either would get swallowed somewhere, and maybe logged if we're lucky.

<sup>1</sup>Why on earth does `MatchError` not extend `java.lang.Error`, and [instead](http://www.scala-lang.org/api/2.11.7/index.html#scala.MatchError) extend `java.lang.Exception`??